packages/client/src/types.ts

```diff
@@ -129,6 +129,12 @@ export interface VercelConfig {
   alias?: string | string[];
   regions?: string[];
   projectSettings?: ProjectSettings;
```

Comment:
Hmm, I wonder if `projectSettings` should be removed since its not available on `vercel.json`

Reply:
if i remember correct it is on there, just undocumented and only used by some of our internal stuff?

Reply:
Is it on vercel.json or part of the API? Because this type represents vercel.json as far as I can tell

Reply:
im not sure. it looks like its been on this interface for a long while though

Reply:
I noticed that `vc dev` uses projectSettings so I guess we should leave it as-is https://github.com/vercel/vercel/blob/960c66584c58ad1e88176796db829514f5db6b15/packages/cli/src/util/dev/server.ts#L567